src/test/java/org/threeten/extra/chrono/TestInternationalFixedChronology.java

```diff
@@ -0,0 +1,1191 @@
+                { 1900, 4, 29 },
+                { 1900, 5, 29 },
+                { 1900, 6, 29 },
+                { 1900, 7, 29 },
+                { 1900, 8, 29 },
+                { 1900, 9, 29 },
+                { 1900, 10, 29 },
+                { 1900, 11, 29 },
+                { 1900, 12, 29 },
+        };
+    }
+
+    @Test(dataProvider = "badDates", expectedExceptions = DateTimeException.class)
+    public void test_badDates(final int year, final int month, final int dom) {
+        InternationalFixedDate.of(year, month, dom);
+    }
+
+    @DataProvider(name = "badLeapDayAndYearDayDates")
+    Object[][] badLeapDayAndYearDayDates() {
+        return new Object[][] {
+                {    1, false, false },
+                {    1, true,  true },
+                {    1, true,  false },
+        };
+    }
+
+    @Test(dataProvider = "badLeapDayAndYearDayDates", expectedExceptions = DateTimeException.class)
+    public void badLeapDayAndYearDayDates(final int year, final boolean isLeapDay, final boolean isYearDay) {
+        InternationalFixedDate.of(year, isLeapDay, isYearDay);
+    }
+
+    @Test(expectedExceptions = DateTimeException.class)
+    public void test_chronology_dateYearDay_badDate() {
+        InternationalFixedChronology.INSTANCE.dateYearDay(2001, 0);
+        InternationalFixedChronology.INSTANCE.dateYearDay(2001, -1);
+        InternationalFixedChronology.INSTANCE.dateYearDay(2001, 366);
+        InternationalFixedChronology.INSTANCE.dateYearDay(2001, 999);
+
+        InternationalFixedChronology.INSTANCE.dateYearDay(2000, 0);
+        InternationalFixedChronology.INSTANCE.dateYearDay(2000, -1);
+        InternationalFixedChronology.INSTANCE.dateYearDay(2000, 367);
+        InternationalFixedChronology.INSTANCE.dateYearDay(2000, 999);
+    }
+
+    //-----------------------------------------------------------------------
+    // InternationalFixedDate.getDayOfYear
+    //-----------------------------------------------------------------------
+    @Test
+    public void test_dateYearDay() {
+        checkYearDateForYear(2001, 365);
+        checkYearDateForYear(2004, 366);
+
+        assertEquals(InternationalFixedChronology.INSTANCE.dateYearDay(2001, 365), InternationalFixedDate.of(2001, false, true));
+        assertEquals(InternationalFixedChronology.INSTANCE.dateYearDay(2004, 169), InternationalFixedDate.of(2004, true, false));
+        assertEquals(InternationalFixedChronology.INSTANCE.dateYearDay(2004, 366), InternationalFixedDate.of(2004, false, true));
+    }
+
+    private void checkYearDateForYear(final int year, final int daysInYear) {
+        InternationalFixedDate date = null;
+        int month = 0;
+        int dayOfMonth = 0;
+        int y = 0;
+
+        for (int i = 1; i <= daysInYear; i++) {
+            date = InternationalFixedChronology.INSTANCE.dateYearDay(year, i);
+            assertEquals(date.getDayOfYear(), i);
+
+            if (date.isYearDay() || date.isLeapDay()) {
```

Comment:
...You at least need to check that they return "not a month values", if not specific flag values for each of them.  Having the test `continue` like this when the object being tested returns a value feels wrong, because of that need.